### DISCUSSION TITLE
Password only for accessing files (instead of account login)

### DISCUSSION THREAD

**Discussion by nikster89** [rejetto/hfs#321] [Ideas]
For now I am using 2.3m with very easy access to files via username and password. In HFS 3 (which is amazing btw!) it seems we need to create&select an account for download&view access. (or am I missing something else?)

Suggestion: Is it possible to create a link+password option? Or even maybe  generate an unique shareable link without password?

**Comment by rejetto** [comment]
hey! such kind of links will be possible, but still don't know when
